style.css

```diff
@@ -404,3 +409,233 @@ main {
   padding: 0;
   padding-top: 10px;
 }
+
+.top-nav {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  body {
+    position: relative;
+
+    /* margin: 0; */
+  }
+
+  .desktop-introduction .desktop-green-h1 {
+    display: none;
+  }
+
+  .main-section {
+    background-image: url(desktop-images/main.svg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    padding: 60px 0;
+    justify-content: center;
+    width: 110vh;
+  }
+
+  .social-media {
+    margin-top: 6rem;
+  }
+
+  .top-nav {
+    display: flex;
+    list-style: none;
+    gap: 10px;
+    text-align: center;
+    margin: auto;
+    padding-bottom: 50px;
+    justify-content: center;
+    color: #172b4d;
+  }
```

Comment:
- [ ] Great job on working on the requested changes from your previouse reviewer, according to your previouse reviewer your website is not responsive yet .need to adjust the width after you have align the navigations items to center.
- [ ]  kindly adjust the width and ensure  your website to be more responsiv on all mobile devices ,I would kindly suggest you to Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels for proper accessibilty and user friendly.


https://github.com/Young-Gaius/portfolio-setup/assets/92458236/629c8974-1acc-4b2c-abf0-d6decee772aa